### ISSUE TITLE
[BUG] Moosync fails to open when installed via Choco on Windows

### ISSUE THREAD

**Issue by Marcus5408** [Moosync/Moosync#194]
**Describe the bug**
When installed via Choco on Windows, Moosync presents a JavaScript error when launched.

**To Reproduce**

1. Open an Administrator terminal window
2. Run `choco install moosync`
3. Wait for install to finish
4. Launch Moosync

**Expected behavior**
Moosync launches and is able to play music.

**Screenshots**
![image](https://user-images.githubusercontent.com/46655509/211171731-f45bd67a-5d66-40f9-bbd3-409030f7c07a.png)

- OS: Windows 11 Pro Insider Preview Version 22H2, Build 25267.1000
- Moosync Version: v6.0.0

**Additional context**
Image transcription:
```
A JavaScript error occurred in the main process
Uncaught Exception:
SqliteError: duplicate column name: playlist_path
  at Database.exec (C:\Program Files\Moosync\resources\app.asar\node_modules\better-sqlite3\lib\methods\wrappers....:14)
  at u.exec (C:\Program Files\Moosync\resources\app.asar\index.js:2:412028)
  at u.migrate (C:\Program Files\Moosync\resources\app.asar\index.js:2:416726)
  at u.connection (C:\Program Files\Moosync\resources\app.asar\index.js:2:411825)
  at u.function (C:\Program Files\Moosync\resources\app.asar\index.js:2:412460)
  at ye.verifySong.registerRegexp (C:\Program Files\Moosync\resources\app.asar\index.js:2:51640)
  at new ye.verifySong (C:\Program Files\Moosync\resources\app.asar\index.js:2:49670)
  at new <anonymous> (C:\Program Files\Moosync\resources\app.asar\index.js:2:49452)
  at 31733 (C:\Program Files\Moosync\resources\app.asar\index.js:2:49448)
  at _webpack_require_ (C:\Program Files\Moosync\resources\app.asar\index.js:8:724206)
```

**Comment by Ovenoboyo** [comment]
Had you installed Moosync from somewhere else before?

**Comment by Ovenoboyo** [comment]
The fix is simple, you'll need to head over to ```C:\Users\<your_username>\AppData\Roaming\moosync\databases``` and delete ```songs.db``` (This will clear all the songs in your library). This usually happens when a newer database is changed by an older version of the app.

**Comment by Marcus5408** [comment]
> Had you installed Moosync from somewhere else before?

No, I've only installed via Choco

> The fix is simple, you'll need to head over to `C:\Users\<your_username>\AppData\Roaming\moosync\databases` and delete `songs.db` (This will clear all the songs in your library). This usually happens when a newer database is changed by an older version of the app.

After deleting `songs.db` I get a different JavaScript error:
![image](https://user-images.githubusercontent.com/46655509/211203345-137dad3b-8fca-4dca-ab79-f6d6296b1b5a.png)

**Comment by Ovenoboyo** [comment]
Oh, you'll need to restart the app for this (You might need to end the current process from the task manager).

This was fixed in v7.0.0+ but unfortunately it takes around 15+days for new versions to be updated on chocolatey.

**Comment by Marcus5408** [comment]
Thanks! This managed to solve my issue and Moosync now launches properly.
